geopandas/io/tests/test_sql.py

```diff
@@ -292,6 +294,20 @@ def test_read_postgis_binary(self, connection_spatialite, df_nybb):
         df = read_postgis(sql, con, geom_col=geom_col)
         validate_boro_df(df)
 
+    def test_read_postgis_chunksize(self, connection_postgis, df_nybb):
+        """Test chunksize argument"""
+        chunksize = 10
```

Comment:
```suggestion
        chunksize = 2
```

(since nybb has only 5 rows)